qiskit/transpiler/passes/layout/vf2_layout.py

```diff
@@ -154,6 +154,11 @@ def run(self, dag):
             if len(cm_graph) == len(im_graph):
                 chosen_layout = layout
                 break
+            # If there is no error map avilable we can just skip the scoring stage as there
+            # is nothing to score with, so any match is the best we can find.
+            if not self.avg_error_map:
+                chosen_layout = layout
+                break
             layout_score = vf2_utils.score_layout(
```

Comment:
Yeah, that's a good point if the score is 0 we should exit fast here since nothing will be that score. It could also happen if all the local ops are ideal.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/8978/commits/b853d2cf651cd22a1ad87dee200198194e43fc12